src/az-purview/deploy/terraform/modules/virtual-network/main.tf

```diff
@@ -0,0 +1,33 @@
+terraform {
```

Comment:
There is an existing virtual network module at src/common/infrastructure/terraform/modules/virtual-network/main.tf. Is there an opportunity to leverage the existing module instead of creating a new module?

Reply:
Same as above, @promisinganuj any thoughts?

Reply:
I have raised an issue to refactor the terraform modules #30